QamarDeen/QamarDeen/Model/Generated/Day+CoreDataProperties.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Day+CoreDataProperties.swift
+//  QamarDeen
+//
+//  Created by Mazyad Alabduljaleel on 12/25/16.
+//  Copyright © 2016 Batoul Apps. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+
+extension Day {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<Day> {
+        return NSFetchRequest<Day>(entityName: "Day");
+    }
+
+    @NSManaged public var hijriDate: NSDate?
```

Comment:
I am maintaining a newspapers app, which stores multiple issues based on dates. Basically, I use a string "YYYYMMDD", which allows for simple querying/sorting and is very natural to a newspapers app. I also see your concern here, and willing to stick with `NSDate` approach.